taxi/forms.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",
+                                                 "first_name",
+                                                 "last_name")
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) == 8 \
+                and license_number[:3].isupper() \
+                and license_number[-5:].isdigit():
+            return license_number
+        else:
+            raise ValidationError("Number should have 3 uppercase letters "
```

Comment:
it's not a true description!) The first 3 letters should be uppercase and the last 5 characters should be digits. And maybe it would be better to specify each error separately?

Reply:
Added some more description